src/main/java/com/redhat/labs/lodestar/resource/EngagementResource.java

```diff
@@ -264,6 +266,19 @@ public Engagement put(@PathParam("id") String uuid, @Valid Engagement engagement
 
     }
 
+    @HEAD
+    @SecurityRequirement(name = "jwt", scopes = {})
+    @Path("/subdomain/{subdomain}")
+    @APIResponses(value = {
+            @APIResponse(responseCode = "401", description = "Missing or Invalid JWT"),
+            @APIResponse(responseCode = "409", description = "Subdomain is already in use"),
+            @APIResponse(responseCode = "200", description = "Engagement resource found and metadata returned in headers") })
+    public Response uniqueSubdomain(@PathParam("subdomain") String subdomain) {
```

Comment:
should probably use SC_OK if using SC_CONFLICT

Reply:
updated to use SC_OK